src/public/main.js

```diff
@@ -1,54 +1,65 @@
+(function($, Handlebars) {
+  // TODO(pascal): get a real number
+  var NUMBER = '678-RESISTS (678-737-4787)'
 
+  /**
+   * CampaignView - view class which consumes a data object and a template id
+   * provides a renderTo method for rendering the view into a node
+   *
+   * @param  {object} options
+   * @return {view} view instance
+   */
+  function CampaignView(options) {
+    if (!options.data) throw ('CampaignView requires data')
+    if (!options.templateId) throw ('CampaignView requires a templateId')
 
-/**
- * CampaignView - view class which consumes a data object and a template id
- * provides a renderTo method for rendering the view into a node
- *
- * @param  {object} options
- * @return {view} view instance
- */
-function CampaignView(options) {
-  if (!options.data) throw ('CampaignView requires data')
-  if (!options.templateId) throw ('CampaignView requires a templateId')
+    this.data = options.data
 
-  this.data = options.data
+    // store source template once
+    var source = $(options.templateId).html()
 
-  // store source template once
-  var source = $(options.templateId).html()
+    // create render method from template source
+    this.render = Handlebars.compile(source)
+  }
+
+  CampaignView.prototype = {
 
-  // create render method from template source
-  this.render = Handlebars.compile(source)
-}
+    // generate html using handlebars render method
+    toHtml: function toHtml() {
+      return this.render(this.data)
+    },
 
-CampaignView.prototype = {
+    // insert rendered html into a dom node
+    renderTo: function renderTo(selector) {
+      $(selector).html(this.toHtml())
+    }
 
-  // generate html using handlebars render method
-  toHtml: function toHtml() {
-    return this.render(this.data)
-  },
+  }
 
-  // insert rendered html into a dom node
-  renderTo: function renderTo(selector) {
-    $(selector).html(this.toHtml())
+  // campaign data object, used on both pages
+  var inauguration = {
+    id: 'ready',
+    name: '(Anti) Inauguration',
+    number: NUMBER,
+    description: 'Or, fill out the form below to stay connected with the movement. Already public pressure has reversed the Republican Party’s attempt to gut the ethics committee, but we cannot back down now. We are the majority. We are the resistance.',
   }
 
-}
+  var view = new CampaignView({
+    templateId: '#campaign-info-template',
+    data: inauguration
+  })
+
+  view.renderTo('.campaign-info')
 
-// campaign data object, used on both pages
-var inauguration = {
-  id: 'ready',
-  name: '(Anti) Inauguration',
-  number: NUMBER,
-  description: 'Or, fill out the form below to stay connected with the movement. Already public pressure has reversed the Republican Party’s attempt to gut the ethics committee, but we cannot back down now. We are the majority. We are the resistance.',
-}
+  // set the campaign in the form input
+  $('#campaign').val(inauguration.id)
+})(window.jQuery, window.Handlebars)
```

Comment:
main.js is shared by index.html and success.html, so this page-specific JS needs to be in a script tag in index.html where it was, not here. (this is a very basic setup until we grow into the need for something more complicated.)